README.adoc

```diff
@@ -249,6 +395,15 @@ go get -u github.com/kisielk/errcheck
 | `make docker_push_master`     | Pushes master image to Docker image hosting repository
 |===
 
+
+=== Using Postman to test the REST API
+
+You can install the https://www.getpostman.com/[Postman] tool which will be useful to call and test the REST API endpoints of this server.
+Following an image to show how it works.
```

Comment:
I'm not sure we need this line. 
I think it can either be removed or added to the above line. Something like 
..... endpoints of this server (See image below).

Reply:
As per my comment above, I suggest removing the image.

Reply:
It was removed